source/code/plugins/change_tracking_runner.rb

```diff
@@ -12,14 +12,15 @@
 class ChangeTrackingRunner 
 	CHANGE_TRACKING_FILE = ARGV[0] 
 	CHANGE_TRACKING_STATE_FILE = CHANGE_TRACKING_FILE + ".hash"
+        CHANGE_TRACKING_INVENTORY_STATE_FILE = CHANGE_TRACKING_FILE + "inventory.hash"
```

Comment:
Please use standardized indentation; the space on this line does not match the line above.